### ISSUE TITLE
Authentication issue.

### ISSUE THREAD

**Issue by sajmoon** [stavro/arc#3]
I get this error:
```
 {:aws_error, {:http_error, 400, 'Bad Request', "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error>
<Code>InvalidRequest</Code><Message>The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256.</Message>
<RequestId>C27DE9ADAC91C66F</RequestId>
```

As you can see there is a problem with the authentication. erlcloud is migrating now to what they refer to as `sign_v4`. You can see the issue here. https://github.com/alertlogic/erlcloud/issues/11

Some of the modules have been migrated, including s3.(https://github.com/alertlogic/erlcloud/commit/12ca9a99701750fd4ab44aea345f1e6cbd77fb8d)

The problem might be because my bucket is in the wrong region (Frankfurt). 
http://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html

**Comment by stavro** [comment]
Thank you - Arc should be still working fine in the US Regions... But I will work on supporting the new signature ASAP.

**Comment by stavro** [comment]
Closed via 277c9957c79652e4f94fd2e5608a57de94f6e40f.  Please reopen if you find additional issues.
